django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -341,7 +341,7 @@ input[type="submit"], button {
         background-position: 30px 12px;
     }
 
-    ul.messagelist li.error {
+    ul.messagelist li.debug, ul.messagelist li.info, ul.messagelist li.error {
```

Comment:
It feels quite strange to me that we have this with what's above and a special case for the warning, would be nice if there was a way to clean this up

Reply:
This is probably caused by the fact that the yes/no icons have a bit of padding in the svg itself, while the alert icon does not have it. Adding the padding to the alert svg should make it possible to simplify the css. What do you think about that?

Reply:
I think that's a good idea to me, we might need to check if we need to change anything else (keep this in a separate commit)